vllm/engine/multiprocessing/engine.py

```diff
@@ -95,8 +100,10 @@ def __init__(self,
         self.input_socket.bind(f"{ipc_path}{IPC_INPUT_EXT}")
 
         # Send output stream back to client.
+        # TODO(robertgshaw2): this currently uses the same path as
```

Comment:
I think this should be okay, but I want to double check. If this is a problem, we can convert this to a separate "error" socket and poll on the error and output socket from the `MQLLMEngineClient`